.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug report
+description: Encountered a clear bug or crash with logs backing it? Fill it up!
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+
+        And also search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
```

Comment:
Make this into a bullet point along with the other two. The way it's structure right now will make people skip this step, even though it's likely the most important of all.

Reply:
Also... Can we not suggest that people respond with reactions? One problem with issues is that there's no way to get updates unless they're commented on since there's no upvote process. Based on that, I think it's better to have them bump the thread even if that means spam.

Reply:
Could also suggest to 'Subscribe' to receive notifications for it.

Reply:
> Can we not suggest that people respond with reactions?

Most issue bumps present zero value and give zero visible reasoning as to why an issue should be reprioritised other than "this person just stumbled upon this issue and figured they should remind us that it's still open". I'd prefer they use reactions for that. But I realise that we've been telling people not to bump and they've been (sporadically) doing it anyway so I don't much care anymore.

Reply:
The thing is that the only way for us to notice reactions is by stumbling on the thread ourselves or always sort by "most reacted" which is not even feasible since you would have to choose which reaction to compare. 

I'm honestly neutral about bump comments, but the bad part is how it would flood the issue thread with extra unnecessary comments. I dunno.

Reply:
I'm personally okay with that. I don't need a notification every time someone wants to remind me that an open issue is open (unless there is a valid reason to reprioritise). But I *do* want to sometimes sort issues by reactions if I want to look at which feature is most requested. I could do the same with comment count I guess, but that's less exact.

Reply:
Of note, you can't sort issues by "most reacted", only discussions.

I think if people feel strongly they will bump regardless of what we say.

Reply:
A potential fix/mitigation for this could be an explanation of priority, something like "we will work on priority:3 issues after XYZ is done, priority:2 issues after ABC, etc" or even directly explaining for specific issues "this affects too few users/is too niche to work on for now, we'll address it once _____". 

People sometimes bump because they feel like an issue is being ignored/forgotten, especially with the issues that stay open for like a year+, even if they aren't forgotten. Having a more rigorous "schedule" for fixes might alleviate those concerns (if you feel that would help). Just an idea, dunno how possible it is to schedule things like that.

Reply:
> People sometimes bump because they feel like an issue is being ignored/forgotten

that's their issue not ours, due to misunderstanding how issue trackers work.

> Having a more rigorous "schedule" for fixes might alleviate those concerns (if you feel that would help).

estimates aren't a solution and i'm not even willing to entertain the thought of estimating things to such a minuscule degree.

Reply:
Both fair points. Just throwing ideas out, see what sticks or inspires.